doc/python/shapes.md

```diff
@@ -682,5 +682,25 @@ fig.update_layout(
 fig.show()
 ```
 
+### Filled Area Chart
```

Comment:
I would move this to the very top, with a little intro saying "there are two ways to draw filled shapes: scatter traces and layout.shapes" with a note that layout.shapes is mostly useful for cartesian 2d subplots, but you can use e.g. scatterpolar and scattergeo and scattermapbox to draw filled shapes on any kind of subplots.

Reply:
You can add a link to the scatterpolar page where we have filled shapes, and to the "filled shapes on mapbox" page that you wrote last year?